code/modules/projectiles/bullet.dm

```diff
@@ -512,8 +512,8 @@ toxic - poisons
 	on_hit(atom/hit, direction, obj/projectile/P)
 		if(slow && ishuman(hit))
 			var/mob/living/carbon/human/M = hit
-			M.changeStatus("slowed", 1.5 SECONDS, optional = 8)
```

Comment:
Intentional that this change increases the raw power of the slow effect? (slowed defaults to 10)

Reply:
It was like that when I got here, officer.

Reply:
The `optional = 8` argument set the slow power to 8 before. Post-change that argument is not present, so the slow power defaults to 10, is what I meant